config_src/infra/FMS1/MOM_interp_infra.F90

```diff
@@ -4,9 +4,11 @@ module MOM_interp_infra
 ! This file is part of MOM6. See LICENSE.md for the license.
 
 use MOM_domain_infra,    only : MOM_domain_type, domain2d
+use MOM_io,              only : axis_info
```

Comment:
These module use statements for MOM_io introduce a backward dependency on code in the src/framework directory and the potential for a predecessor cycle.  There are no other instances of config_src/infra code on src/framework code, and I think that we should keep it that way.  Please consider revising this dependency.

Reply:
This use statement has not created a predecessor cycle between modules.